### PR TITLE
make Slig::HandlePlayerControlled_4B7800() a bit cleaner

### PR DESCRIPTION
**Pull request by mouzedrift**

### CODE REVIEW

**Comment by mouzedrift** [comment]
dont merge yet still need to fix something :oddtism:

**Comment by mouzedrift** [comment]
ah nvm should be good to go i guess

File: Source/AliveLibAE/Slig.cpp [hunk 1]
```diff
@@ -5568,193 +5568,210 @@ __int16 Slig::MainMovement_4B4720()
...
+    {
+        return 0;
+    }
+
+    field_C4_velx = SteppingToStandVelX;
+    field_106_current_motion = eSligMotions::M_SteppingToStand_14_4B8480;
+    return 1;
+}
+
+__int16 Slig::PullLever()
```
Comment (paulsapps) [inline Source/AliveLibAE/Slig.cpp:5632]: return value is pointless, it always returns 1

File: Source/AliveLibAE/Slig.cpp [hunk 2]
```diff
@@ -5568,193 +5568,210 @@ __int16 Slig::MainMovement_4B4720()
...
-            field_C4_velx = (ScaleToGridSize_4498B0(field_CC_sprite_scale) / FP_FromInteger(6));
-            field_106_current_motion = eSligMotions::M_SteppingToStand_14_4B8480;
-            return 1;
-        }
-        else
-        {
-            field_106_current_motion = eSligMotions::M_TurnAroundStanding_5_4B6390;
-            return 1;
-        }
+__int16 Slig::ShootZ()
```
Comment (paulsapps) [inline Source/AliveLibAE/Slig.cpp:5660]: ToShootZ ?

File: Source/AliveLibAE/Slig.cpp [hunk 3]
```diff
@@ -5568,193 +5568,210 @@ __int16 Slig::MainMovement_4B4720()
...
+        field_106_current_motion = eSligMotions::M_ShootZ_42_4B7560;
+        field_108_next_motion = -1;
+        field_12C_timer = sGnFrame_5C1B84 + 60;
+        return 1;
     }
+    return 0;
+}
 
-    if (sInputObject_5BD4E0.isPressed(sInputKey_Left_5550D4))
+__int16 Slig::ShootOrShootZ()
```
Comment (paulsapps) [inline Source/AliveLibAE/Slig.cpp:5674]: dead return value

File: Source/AliveLibAE/Slig.hpp [hunk 4]
```diff
@@ -436,6 +436,16 @@ class Slig : public BaseAliveGameObject
 
     EXPORT __int16 MainMovement_4B4720();
 
+    enum class Direction
```
Comment (paulsapps) [inline Source/AliveLibAE/Slig.hpp:439]: maybe name it MovementDirection ?

**Pull request merged**